greenpithumb/camera.py

```diff
@@ -0,0 +1,27 @@
+class Camera(object):
+    """Wrapper for a Raspberry Pi Camera."""
+
+    def __init__(self, folder_path, local_clock, camera):
+        """Creates a new camera wrapper instance.
+
+        Args:
+            folder_path: String of the path name of the folder where pictures
+                will be stored.
+            local_clock: Local clock interface.
+            camera: Camera interface.
+        """
+        self._folder_path = folder_path
+        self._local_clock = local_clock
+        self._camera = camera
+
+    def capture(self):
+        """Captures an image from the camera."""
```

Comment:
Suggests "Captures an image from the camera and saves it to the destination folder." Maybe rename the function to something like `save_photo` to make the purpose a little clearer.